### PR TITLE
fix https://github.com/w3c/jlreq/issues/67

### PR DESCRIPTION
**Pull request by macnmm**
…change “ruby letter(s)” to “ruby character(s)”; correct typo “it’s” t…o “its”; change “go over” to “overhang”

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/jlreq/pull/69.html" title="Last updated on Jun 19, 2019, 1:05 AM UTC (f8ac2d3)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/jlreq/69/8da8f10...f8ac2d3.html" title="Last updated on Jun 19, 2019, 1:05 AM UTC (f8ac2d3)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -3416,17 +3416,17 @@ <h4 id="usage_of_ruby">
...
               <p its-locale-filter-list="ja" lang="ja">日本語表記では，複数の<a class="characterClass" href="#cl-19">漢字等（cl-19）</a>で構成された<a href="#term.compound-word" class="termref">熟語</a>をよく使う．この熟語を構成する個々の<a class="characterClass" href="#cl-19">漢字等（cl-19）</a>について，音読み又は訓読みの読み方を示す<a class="characterClass" href="#cl-15">平仮名（cl-15）</a>のルビを付ける（このようなルビの使用例は多い）．このような<span class="index" id="d5e8806"><span class="index" id="d5e8807">熟語に付けるルビ</span></span>について，親文字とルビ文字との対応には，2つの方法がある．</p>
               <ol>
                 <li id="id219">
-                  <p its-locale-filter-list="en" lang="en"><a href="#term.mono-ruby" class="termref2nd">Mono-ruby</a>. Ruby letters are attached to each base <a class="characterClass" href="#cl-19">ideographic character (cl-19)</a>, similarly to the previous section (see [[[#fig2_3_3]]]).</p>
+                  <p its-locale-filter-list="en" lang="en"><a href="#term.mono-ruby" class="termref2nd">Mono-ruby</a>. ruby characters are attached to each base <a class="characterClass" href="#cl-19">ideographic character (cl-19)</a>, similarly to the previous section (see [[[#fig2_3_3]]]).</p>
```
Comment (himorin) [inline index.html:3423]: capitalize the first character?

File: index.html [hunk 2]
```diff
@@ -3416,17 +3416,17 @@ <h4 id="usage_of_ruby">
...
                   <p its-locale-filter-list="ja" lang="ja">前項と同様に，個々の<a class="characterClass" href="#cl-19">漢字等（cl-19）</a>（親文字）の1文字ごとにそのルビを対応させる方法．つまり，<a href="#term.mono-ruby" class="termref2nd">モノルビ</a>としてルビを処理する方法である（[[[#fig2_3_3]]]）．</p>
                   <figure id="fig2_3_3"> 
-      <img its-locale-filter-list="en" src="images/img2_3_3.png" alt="Example of mono-ruby method. Ruby letters are attached to each base ideographic character in a compound word." width="117" height="369">
+      <img its-locale-filter-list="en" src="images/img2_3_3.png" alt="Example of mono-ruby method. ruby characters are attached to each base ideographic character in a compound word." width="117" height="369">
```
Comment (himorin) [inline index.html:3426]: capitalize the first character?

File: index.html [hunk 3]
```diff
@@ -3416,17 +3416,17 @@ <h4 id="usage_of_ruby">
...
       <img its-locale-filter-list="ja" lang="ja" src="images_ja/img2_3_3.png" alt="熟語に付くルビをモノルビで処理した例" width="101" height="369">
                     <figcaption>
-        <span its-locale-filter-list="en" lang="en">Example of mono-ruby method. Ruby letters are attached to each base ideographic character in a compound word.</span>
+        <span its-locale-filter-list="en" lang="en">Example of mono-ruby method. ruby characters are attached to each base ideographic character in a compound word.</span>
```
Comment (himorin) [inline index.html:3429]: capitalize the first character?

File: index.html [hunk 4]
```diff
@@ -3440,20 +3440,20 @@ <h4 id="usage_of_ruby">
...
                   <p its-locale-filter-list="ja" lang="ja">個々の<a class="characterClass" href="#cl-19">漢字等（cl-19）</a>の読み方を示すと同時に，熟語単位での配置も考慮して配置する方法（[[[#fig2_3_4]]]）．この方法は<span class="index" id="d5e8948"><a href="#term.jukugo-ruby" class="termref">熟語ルビ</a></span>とよばれている．熟語ルビは，熟語を構成する<a class="characterClass" href="#cl-19">漢字等（cl-19）</a>の1文字1文字の読み方を示すとともに，熟語としてのまとまりを持っているので，ルビもまとまりとして読める必要がある．熟語ルビは，熟語としてのまとまりを重視した配置方法といえる．</p>
                   <figure id="fig2_3_4"> 
-      <img its-locale-filter-list="en" src="images/img2_3_4.png" alt="Example of jukugo-ruby method. Ruby letters are attached to groups of ideographic characters in compound words." width="242" height="360">
+      <img its-locale-filter-list="en" src="images/img2_3_4.png" alt="Example of jukugo-ruby method. ruby characters are attached to groups of ideographic characters in compound words." width="242" height="360">
```
Comment (himorin) [inline index.html:3443]: capitalize the first character?

File: index.html [hunk 5]
```diff
@@ -3440,20 +3440,20 @@ <h4 id="usage_of_ruby">
...
       <img its-locale-filter-list="ja" lang="ja" src="images_ja/img2_3_4.png" alt="熟語に付くルビを熟語ルビで処理した例" width="250" height="360">
                     <figcaption>
-        <span its-locale-filter-list="en" lang="en">Example of jukugo-ruby method. Ruby letters are attached to groups of ideographic characters in compound words.</span>
+        <span its-locale-filter-list="en" lang="en">Example of jukugo-ruby method. ruby characters are attached to groups of ideographic characters in compound words.</span>
```
Comment (himorin) [inline index.html:3446]: capitalize the first character?

**Review by himorin — CHANGES_REQUESTED** [review]

File: index.html (hunk 5 above)
Comment (r12a) [reply index.html:3446]: I'd say, yes, capitalise the r in ruby in all cases you noticed.

File: index.html (hunk 1 above)
Comment (macnmm) [reply index.html:3423]: I need a better tool for editing this... Sorry for the typos!

**Comment by macnmm** [comment]
OK, I noticed a discrepancy in the description and logic of overhang of adjacent punctuation, requiring a change in both the English and the Japanese. I believe this is the correct behavior (it is what we put in InDesign, anyway). @himorin please confirm?

**Comment by himorin** [comment]
@kidayasuo could you check the last commit? f8ac2d3 changes Japanese text, the last part is modified to '前の空き量以外に掛けてはならない．'.
note, 7ad864a is rebase to the master/HEAD.

**Comment by macnmm** [comment]
Closing this mess. Will open PR to address ^& and dress the other issues in other PRs.

**Pull request closed without merging**